app/adapters/application.js

```diff
@@ -3,15 +3,19 @@ import DS from 'ember-data';
 import ENV from '../config/environment';
 
 export default DS.RESTAdapter.extend(DS.BuildURLMixin, {
+    session: Ember.inject.service(),
     namespace: 'api/v2',
     host: ENV.apiBaseUrl,
     headers: {
         Accept: 'application/json'
     },
     ajax(url, method, hash) {
+        // trailing slash required for POST requests
+        url = url + '/';
```

Comment:
You'll probably need to do a regex here. This will break on urls like `/api/v2/foo/` and `/api/v2/foo?bar=baz`